pkg/test/fake/fake.go

```diff
@@ -49,7 +49,7 @@ var subresource_list = []client.Object{
 	&hiveintv1alpha1.FakeClusterInstall{},
 }*/
 
-func NewFakeClientBuilder() *fake.ClientBuilder {
+func NewScheme() *runtime.Scheme {
```

Comment:
So let's make this func private.

Or possibly better yet, just run this code in an init() to initialize hive_scheme.